gluonnlp/data/dataloader.py

```diff
@@ -197,4 +126,5 @@ def _same_process_iter():
 
         # multi-worker
         return _ShardedMultiWorkerIter(self._num_workers, self._dataset,
```

Comment:
Is it possible to remove `_ShardedMultiWorkerIter` and just call `_MultiWorkerIter` with the appropriate worker loop?

Reply:
Oh, you are right. We can do it.